recipes/javafx/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "javafx-sdk" %}
+{% set version = "11.0.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/javafxports/openjdk-jfx/archive/11.0.4+1.tar.gz
+  sha256: f03d16adae701406ff9e1fc05e461a327ca4f97b5bf18b0d727037bb425ec25a
+
+build:
+  number: 0
+  skip: true  # [win]
+  missing_dso_whitelist:
+    - '*'
```

Comment:
That is required, as otherwise the packaging fails.

> ERROR (javafx,lib/javafx-11.0.4/lib/libglassgtk2.so): $RPATH/libpango-1.0.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=75538&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c

Reply:
I see now, it's a real issue and not a false positive. You shouldn't need to change `PKG_CONFIG_PATH` in your build script. Instead `freetype`, `pango`, `gtk2` should be in the `host` dependencies instead of `build`. There might also be others that are in the wrong section but unsetting `PKG_CONFIG_PATH` should expose them.

Reply:
I think now it should be ok ?